### ISSUE TITLE
Problem with dial: dial tcp 172.22.0.2:11334: getsockopt: connection refused.

### ISSUE THREAD

**Issue by ictinc** [jeboehm/docker-mailserver#67]
Hi there,
After going through the instructions I'm now trying to create my first user with the command:
`bin/production.sh` run --rm web setup.sh`

Unfortunately this results in the following (repeated) error message:
`Problem with dial: dial tcp 172.22.0.4:80: getsockopt: connection refused. Sleeping 1s`

When I try to see the logs logs being created on the web container with:
`docker logs -f container`

I'm presented with a continuous flow of the same error message:
`2019/06/05 20:48:21 Problem with dial: dial tcp 172.22.0.3:25: getsockopt: connection refused. Sleeping 1s
2019/06/05 20:48:21 Problem with dial: dial tcp 172.22.0.2:11334: getsockopt: connection refused. Sleeping 1s`

While these two IP addresses are responding to pings a telnet to these IPs on their respective ports (25 and 11334) give a connection refused.
Also issuing a curl command to the web container also results in a connection refused.

`docker `ps`` shows all containers as running however, the filter, web, mta and virus container have the Health status to starting.

I'm not sure how to go from here. I hope you have any suggestions.

Kind regards

**Comment by jeboehm** [comment]
Hey @ictinc ,
hard to say what's wrong here. It seems that not all containers are started.
Please give the output of `bin/production.sh ps`. If there is a container that is exited or aborted, you can gather its logs by executing `bin/production.sh logs SERVICENAME`. SERVICENAME is something like mda, mta or filter.

**Comment by infostreams** [comment]
I'm fairly sure that this is because you have other containers or processes running that are already bound to one of the ports that the mailserver is trying to bind to. In my case, I already had a webserver open on port 80. Once I shut that process down, and waited a bit, this error disappeared.

**Comment by infostreams** [comment]
Ok, I got the error again (and again), and by looking at the logs it turned out that it was caused by the virus container that somehow refused to start correctly. It complained that it was out of date. That then lead to a cascade of log entries from other containers that depend on the virus scanner. So, I disabled the virus scanner. For my (very specific) purposes I don't need it anyway. After that all is fine and dandy.
